samples/net/sockets/echo/README.rst

```diff
@@ -76,3 +76,15 @@ To test:
 
```

Comment:
Usually for samples, the commit subject starts with "samples: net: ...."

Reply:
OK